OpenRA.Game/Server/Server.cs

```diff
@@ -735,7 +760,7 @@ void HandleSyncOrder(int frame, byte[] packet)
 			}
 		}
 
-		public void DispatchOrdersToClients(Connection conn, int frame, byte[] data)
+		public void DispatchOrdersToClients(Connection conn, int frame, byte[] data, bool buffer = false)
```

Comment:
The new parameter is unused?

Reply:
Please use "is"/"has"/"should"/"can" prefixes for bools as per C# conventions.

Reply:
Buffer is already a verb, so this should be fine?